lib/nylas/model/attributable.rb

```diff
@@ -7,7 +7,11 @@ def self.included(model)
       end
 
       def initialize(**initial_data)
-        initial_data.each do |attribute_name, value|
+        merge(**initial_data)
+      end
+
+      protected def merge(**data)
```

Comment:
This doesn't do what I would call merging afaict - would `assign_attributes` or something make more sense?